php-plugin/src/test/resources/checks/AvoidSQLRequestInLoop.php

```diff
@@ -2,73 +2,72 @@
 
 class AvoidFullSQLRequest
 {
-	private $DbUser = 'user';
-	private $DbName = 'name';
-	private $DbPass = 'pass';
-	private $DbHost = 'host';
-	private $Query = 'SELECT * FROM Table';
-	private $OtherQuery = 'SELECT name FROM User';
-	private $Connection;
+    private $dbUser = 'user';
+    private $dbName = 'name';
+    private $dbPass = 'pass';
+    private $dbHost = 'host';
+    private $query = 'SELECT * FROM Table';
+    private $otherQuery = 'SELECT name FROM User';
+    private $connection;
+    public function launchSQLRequest($someCondition) {
```

Comment:
PSR-12 also asks to put braces on newlines, I don't know if you want to follow this guideline?
https://www.php-fig.org/psr/psr-12/#41-extends-and-implements